lib/components/ConfirmButton.vue

```diff
@@ -32,11 +32,11 @@ import uniqueId from 'lodash/uniqueId'
 import { faTimes } from '@fortawesome/free-solid-svg-icons/faTimes'
 import { BTooltip } from 'bootstrap-vue/esm/components/tooltip/tooltip'
 
-import { library } from './Fa'
+import { library, default as Fa } from './Fa'
```

Comment:
```suggestion
import Fa from './Fa'
```
To avoid importing twice the library object, maybe it could be interesting to just use Fa and then line 159 use`Fa.library` (

Reply:
As far as I know, we cannot use `Fa.library`

Reply:
Ok so all of the other comments are pointless ^^

Reply:
😵